requirements.in

```diff
@@ -0,0 +1,2 @@
+aiohttp
+sqlalchemy
```

Comment:
in that case, get rid of postgres in `docker-compose.yml` for now, or implement postgres (preferred)